docker-compose.yml

```diff
@@ -20,7 +20,18 @@ services:
     environment:
       MYSQL_ROOT_PASSWORD: password
     restart: unless-stopped
- 
+
+  mongodb:
+    image: mongo:latest
```

Comment:
should we fix this at a version we know works?

Reply:
Done!